extensions/grub.sh

```diff
@@ -116,7 +116,7 @@ pre_umount_final_image__install_grub() {
 	rm -f "${MOUNT}"/etc/default/grub.d/99-armbian-host-side.cfg
 
 	local root_uuid
-	root_uuid=$(blkid -s UUID -o value "${LOOP}p1") # get the uuid of the root partition, this has been transposed
+	root_uuid=$(blkid -s UUID -o value "${LOOP}p${uefipart}") # get the uuid of the root partition, this has been transposed
```

Comment:
`uefipart` ?

`/boot/grub.cfg` is on `root`. When use `boot partition`, it's on `/boot`

Reply:
This is a fix for a fixed name, which may be different (depends on the variable). See the source code

Reply:
the root_uuid is used to point to the... ROOT, not the ESP partition, so this makes no sense, unless they're the same, which makes no sense either.

Reply:
> the root_uuid is used to point to the... ROOT, not the ESP partition, so this makes no sense, unless they're the same, which makes no sense either.

If you want to have an error in the code, it's your right. I will remove this fix from your extension.